vdp/mgmt/v1alpha/mgmt_public_service.proto

```diff
@@ -39,15 +39,15 @@ service MgmtPublicService {
   // a GetAuthenticatedUserResponse message.
   rpc GetAuthenticatedUser(GetAuthenticatedUserRequest) returns (GetAuthenticatedUserResponse) {
     option (google.api.http) = {
-      get : "/v1alpha/user"
+      get : "/v1alpha/users/me"
     };
   }
 
   // UpdateAuthenticatedUser method receives a UpdateAuthenticatedUserRequest message and returns 
   // a UpdateAuthenticatedUserResponse message.
   rpc UpdateAuthenticatedUser(UpdateAuthenticatedUserRequest) returns (UpdateAuthenticatedUserResponse) {
```

Comment:
```suggestion
  rpc LookUpAuthenticatedUser(LookUpAuthenticatedUserRequest) returns (LookUpAuthenticatedUserResponse) {
```

Reply:
After second thought, following AIP-136, 
> The verb in the name should not contain any of the standard method verbs ([Get](https://google.aip.dev/131), [List](https://google.aip.dev/132), [Create](https://google.aip.dev/133), [Update](https://google.aip.dev/134), [Delete](https://google.aip.dev/135)).

I have made the change in the latest commit 
- refactor `GetAuthenticatedUser` to `QueryAuthenticatedUser`, instead of `LookUpAuthenticatedUser`, since we use LookUp for querying resource by UUID
- refactor `UpdateAuthenticatedUser` to `PatchAuthenticatedUser`

Please help check.